### PR TITLE
Fix accessibility related issues in enrolment tab ccx dashboard

### PR DESCRIPTION
**Pull request by amir-qayyum-arbisoft**
HI

In this PR I fixed accessibility related issues on enrolment tab inside ccx dashboard. 

@ahsan-ul-haq @muzaffaryousaf @symbolist

### CODE REVIEW

File: lms/templates/ccx/enrollment.html [hunk 1]
```diff
@@ -45,13 +45,13 @@
...
 
-<div class="member-lists-management" style="float:left;width:50%">
+<div class="member-lists-management" style="float:left;width:50%" aria-live="polite" role="region">
   <form method="POST" action="ccx_manage_student">
   <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
   <div class="auth-list-container active">
     <div class="member-list-widget">
-      <div class="member-list">
-        <h2> ${_("Student List Management")}</h2>
+      <div class="mem   ber-list">
```
Comment (ahsan-ul-haq) [inline lms/templates/ccx/enrollment.html]: shouldn't it be member-list?

**Comment by symbolist** [comment]
Looks okay to me. :+1:.

Just get a thumbs up from @ahsan-ul-haq.

File: lms/templates/ccx/enrollment.html (hunk 1 above)
Comment (amir-qayyum-arbisoft) [inline lms/templates/ccx/enrollment.html]: fixed

**Comment by ahsan-ul-haq** [comment]
:+1:

**Comment by amir-qayyum-arbisoft** [comment]
open

**Pull request closed without merging**